src/lib.rs

```diff
@@ -447,9 +439,11 @@ fn compress_chunks_parallel(
     debug_assert!(input.len() <= MAX_SIMD_DEGREE * CHUNK_LEN);
 
     let mut chunks_exact = input.chunks_exact(CHUNK_LEN);
-    let mut chunks_array = ArrayVec::<[&[u8; CHUNK_LEN]; MAX_SIMD_DEGREE]>::new();
-    for chunk in &mut chunks_exact {
-        chunks_array.push(array_ref!(chunk, 0, CHUNK_LEN));
+    let mut chunks_array = [&[0u8; CHUNK_LEN]; MAX_SIMD_DEGREE];
```

Comment:
`CHUNK_LEN = 64` but `[T; 64]: Default` is not true until const-generics land; so this has been expanded to just an array and associated length.